Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs

```diff
@@ -130,8 +133,37 @@ public void LoadImage()
                 }
             }
 
+			public void LoadShopImage()
+			{
+				var url = ShopURL;
+				if (!string.IsNullOrEmpty(url))
+				{
+					//Debug.Log(url + " and " + DisplayAd);
+					ImageDownloader.New().Download(url,
+						result =>
+						{
+							shoptexture = result;
+							//CurrentState = State.Loaded;
+
+						},
+						error =>
+						{
+							ScutiLogger.LogError("Failed to load: " + url + " for " + Title);
+							//CurrentState = State.Failed;
+						}
+					);
+				}
+				else
+				{
+#if UNITY_EDITOR
+					ScutiLogger.LogError("No URL for " + this.ToJson());
+#endif
+					shoptexture = null;
```

Comment:
Probably need to call cleanup here instead of null so it gets disposed

Reply:
is  `Destroy(shoptexture)` enough for the cleanup

Reply:
I think we should call CleanUp(shopImage.sprite). Been awhile but I think the sprite may have left garbage collection as well since we were creating that dynamically?  I'm not 100% on that. Destroy could be fine

Reply:
Check the changes, I copied them from the line 170 where the clean up is made